### ISSUE TITLE
Loading Microsoft.NetCore.Analyzers.Runtime.TestForNaNCorrectlyAnalyzer

### ISSUE THREAD

**Issue by verysimplenick** [dotnet/roslyn-analyzers#1331]
#### Analyzer package

Microsoft.NetCore.Analyzers

#### Analyzer

Microsoft.NetCore.Analyzers.Runtime.TestForNaNCorrectlyAnalyzer

#### Repro steps

1. Create any netcore proj
1. See error list

#### Expected behavior

Worked analyzers

#### Actual behavior

Warning in error list:

```
Warning	AD1000	An instance of analyzer Microsoft.NetCore.Analyzers.Runtime.TestForNaNCorrectlyAnalyzer cannot be created from C:\USERS\____CROPED____\APPDATA\LOCAL\MICROSOFT\VISUALSTUDIO\15.0_81F96476\EXTENSIONS\ANPPWXHN.ZXU\Microsoft.NetCore.Analyzers.dll: Exception has been thrown by the target of an invocation..
```

VS 2017 latest preview.

**Comment by dfederm** [comment]
Here's a stack trace:

```
An instance of analyzer Microsoft.NetCore.Analyzers.Runtime.TestForNaNCorrectlyAnalyzer cannot be created from C:\Users\David\.nuget\packages\microsoft.netcore.analyzers\2.3.0-beta1\analyzers\dotnet\cs\Microsoft.NetCore.Analyzers.dll: Exception has been thrown by the target of an invocation..	

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.TypeLoadException: Could not load type 'Microsoft.CodeAnalysis.Semantics.BinaryOperationKind' from assembly 'Microsoft.CodeAnalysis, Version=2.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'.
   at Microsoft.NetCore.Analyzers.Runtime.TestForNaNCorrectlyAnalyzer..ctor()
   --- End of inner exception stack trace ---
   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor, Boolean& bNeedSecurityCheck)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic)
   at System.Activator.CreateInstance(Type type)
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerFileReference.Extensions`1.GetAnalyzersForTypeNames(Assembly analyzerAssembly, IEnumerable`1 analyzerTypeNames, Boolean& reportedError)
-----
System.TypeLoadException: Could not load type 'Microsoft.CodeAnalysis.Semantics.BinaryOperationKind' from assembly 'Microsoft.CodeAnalysis, Version=2.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'.
   at Microsoft.NetCore.Analyzers.Runtime.TestForNaNCorrectlyAnalyzer..ctor()
-----

```

**Comment by dfederm** [comment]
Looks like it's caused by dotnet/roslyn#21407 and fixed by #1308. We just need a new version of the Microsoft.CodeAnalysis.FxCopAnalyzers Nuget I think.

**Comment by mavasani** [comment]
Yes, @dfederm pointed out, this issue should be fixed with latest FXCop analyzers preview package https://dotnet.myget.org/feed/roslyn-analyzers/package/nuget/Microsoft.CodeAnalysis.FxCopAnalyzers/2.6.0-beta1-62231-02 on latest VS2017 15.5 preview bits.

**Comment by verysimplenick** [comment]
@mavasani what about [plugin](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.MicrosoftCodeAnalysis2017)?

**Comment by jinujoseph** [comment]
vsix plugin has not been updated to the latest yet.
We will release an updated plugin in next two weeks.

**Comment by verysimplenick** [comment]
@jinujoseph 26 days went, update for plugin not released.
Any news?

**Comment by 333fred** [comment]
@verysimplenick we're planning on releasing an update to both the nuget and VSIX packages before the end of the year. Thanks for the interest!

**Comment by verysimplenick** [comment]
@333fred, @jinujoseph  one weekend in 2017 remains.
Still waiting... :(

**Comment by mavasani** [comment]
@kuhlenh Can we publish the VSIX after the holidays? Note that on 15.5.1, https://github.com/dotnet/roslyn/issues/23766 still causes issues when both NuGet and VSIX are installed. @heejaechang @jinujoseph When are we releasing the next 15.6 preview which will contain the fix for https://github.com/dotnet/roslyn/issues/23766?

**Comment by jinujoseph** [comment]
This fix will be part of our 15.6 preview 2  release, should be available sometime during the 2'nd/3'rd week of January.
